docs/API/plain_rules.md

```diff
@@ -170,9 +170,17 @@ fields:
     values are truncated to them (a warning log message is generated if such truncation occurs)
 -   **attribute**: _mandatory_, attribute for watch
 -   **reportInterval**: _mandatory_, time in seconds to see an entity as silent (in other words, how much time has to
-    pass since last entity update to consider that entity is in "no signal" situation so the rule is triggered)
+    pass since last entity update to consider that entity is in "no signal" situation so the rule is triggered). In
+    order to not check too old entities there is a _maxTimeDetection_ (perseo checks for the attribute first in the
+    entity configuration, maxTimeDetectionAttr, then in the rule configuration, and lastly, if not found, in the global
+    configuration) to limit this search
+-   **reportIntervalAttr**: _optional_, entity attribute where find value for reportInterval.
+-   **maxTimeDetection**: _optional_, max time in seconds to see an entity as silent (in other words, how much time has
+    to pass since last entity update to not consider that entity is in "no signal" situation due to entity is death (no
+    modified during a long time)
+-   **maxTimeDetectionAttr**: _optional_, entity attribute where find value for maxTimeDetection.
```

Comment:
Maybe this new fields (3) in this kind of rules should be mentioned in CNR?

Reply:
In addition, being all 3 optional attributes, it should be specified what happen when they are not provided. In particular:

* In which attribute is reportInternval when reportInterval is not provided?
* Which value takes the max detection time when maxTimeDetection is not provided? (I guess that the one in PERSEO_NON_SIGNAL_MAX_TIME_DETECTION, but it is not explicitly said here)
* Which attribute is used when maxTimeDetectionAttr is not used? (In fact, I'm not sure that means this attribute... as maxTimeDetection is a field in the rule...)

Reply:
Explained here: https://github.com/telefonicaid/perseo-fe/pull/827/commits/034b91072ed079ed1ad0c7a73770bb46471757cd